eox_tagging/api/v1/serializers.py

```diff
@@ -88,11 +82,16 @@ def create(self, validated_data):
         except ValidationError as e:
             raise serializers.ValidationError({"Tag": _("{}".format(e.message))})
 
-    def __convert_compound_keys(self, ids, object_type):
+    @staticmethod
+    def _convert_compound_keys(ids, object_type):
         """
         Function that converts strings with format: `key1: key2` into a dictionary.
         """
-        target_id = re.split(r':\s', ids)
+        target_id = ids.replace(" ", "").split(":", 1)
```

Comment:
This applies to enrollments too right? shouldn't it be `rsplit` instead? or I'm missing something?

Reply:
I don't get it. Course enrollments use: `username: course_id`

Reply:
My bad, I had it backwards in my mind `course_id: username`.